operator/v1/types_console.go

```diff
@@ -171,6 +175,14 @@ type DeveloperConsoleCatalogCategory struct {
 	Subcategories []DeveloperConsoleCatalogCategoryMeta `json:"subcategories,omitempty"`
 }
 
+// QuickStarts allow cluster admins to customize available quick starts
+type QuickStarts struct {
+	// disabled is a list of names of quick starts that are not shown to the users
```

Comment:
We should make it clear that these are the names ConsoleQuickStart resources. I would probably use the full PascalCase name here and in the comment above to make it clear it's referencing a specific CRD.

Eventually, we'll need to update this to handle the name annotation used for translations, but I would not mention that yet until we've implemented translations for quick starts.

Reply:
@spadgett Updated as per the suggestion.